lib/screens/buy/widgets/payment_converter.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:realunit_wallet/packages/utils/default_assets.dart';
+import 'package:realunit_wallet/screens/buy/cubits/buy_converter/buy_converter_cubit.dart';
+import 'package:realunit_wallet/styles/colors.dart';
+import 'package:realunit_wallet/styles/currency.dart';
+
+class PaymentConverter extends StatelessWidget {
+  const PaymentConverter({
+    super.key,
+    required TextEditingController amountController,
+    required TextEditingController resultController,
+  })  : _amountController = amountController,
+        _resultController = resultController;
+
+  final TextEditingController _amountController;
+  final TextEditingController _resultController;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Padding(
+          padding: EdgeInsets.symmetric(
+            horizontal: 12.0,
+            vertical: 4.0,
+          ),
+          child: Text(
+            S.of(context).you_pay,
+            style: TextStyle(
+              fontSize: 13,
+              fontWeight: FontWeight.bold,
+            ),
+          ),
+        ),
+        Container(
+          decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(8.0),
+            border: Border.all(color: RealUnitColors.neutral300),
+          ),
+          child: ClipRRect(
+            borderRadius: BorderRadius.circular(8.0),
+            child: Row(
+              children: [
+                Expanded(
+                  flex: 3,
+                  child: TextField(
+                    controller: _amountController,
+                    keyboardType: const TextInputType.numberWithOptions(decimal: true),
+                    decoration: const InputDecoration(
+                      border: InputBorder.none,
+                      contentPadding: EdgeInsets.symmetric(
+                        horizontal: 10.0,
+                        vertical: 14.0,
+                      ),
+                    ),
+                    maxLines: 1,
+                    onChanged: (value) => context.read<BuyConverterCubit>().onFiatChanged(value),
+                  ),
+                ),
+                Container(
+                  color: RealUnitColors.neutral300,
+                  width: 1,
+                  height: 52,
+                ),
+                Expanded(
+                  flex: 2,
+                  child: Container(
+                    color: RealUnitColors.neutral50,
+                    padding: const EdgeInsets.only(
+                      top: 8.0,
+                      bottom: 8.0,
+                      left: 10.0,
+                      right: 4.0,
+                    ),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(
+                          Currency.chf.code,
```

Comment:
This should be depending on the user setting. Not sure if the endpoint supports EUR yet so please just add an in  code TODO comment